### PR TITLE
Adding SEO for Webbsite

### PR DESCRIPTION
**Pull request by AtelyPham**
## Summary of changes
_Provide a detailed description of proposed changes._
- Adding SEO for pages in Webbsite app. 

### Proposed area of change
_Put an `x` in the boxes that apply._

- [ ] `apps/bridge-dapp`
- [ ] `apps/stats-dapp`
- [x] `apps/webbsite`
- [ ] `libs/webb-ui-components`

### Reference issue to close (if applicable)
_Specify any issues that can be closed from these changes (e.g. Closes #233)._
- Closes `NaN`

### Screen Recording
_If possible provide a screen recording of proposed change._

https://user-images.githubusercontent.com/60747384/217791142-093fa78a-670b-4965-87d2-3a24e2d3f3d8.mp4

-----
### Code Checklist 
_Please be sure to add .stories documentation if any additions are made to `libs/webb-ui-components`._

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by devpavan04 — APPROVED** [review]

**Comment by AtelyPham** [comment]
Thanks for your review @devpavan04, please also review the chromatic UI whenever you can.

File: apps/webbsite/src/pages/privacy-policy.tsx [hunk 1]
```diff
@@ -4,24 +4,29 @@ import { ExtendedRecordMap } from 'notion-types';
...
-      components={{
-        Header: Heading2,
-        nextImage: Image,
-        nextLink: Link,
-      }}
-      pageCover={<div />} // Keep the content inside the <main></main>
-      recordMap={data}
-    />
+    <>
+      <NextSeo title="Privacy Policy" description="The Webb's Privacy Policy" />
```
Comment (dutterbutter) [inline apps/webbsite/src/pages/privacy-policy.tsx:15]:
```suggestion
      <NextSeo title="Privacy Policy" description="Privacy Policy of Webb" />
```

File: apps/webbsite/src/pages/terms-and-conditions.tsx [hunk 2]
```diff
@@ -1,31 +1,39 @@
...
-        nextImage: Image,
-        nextLink: Link,
-      }}
-      pageCover={<div />} // Keep the content inside the <main></main>
-      recordMap={data}
-    />
+    <>
+      <NextSeo
+        title="Terms and Conditions"
+        description="The Webb's Terms and Conditions"
```
Comment (dutterbutter) [inline apps/webbsite/src/pages/terms-and-conditions.tsx:18]:
```suggestion
        description="Terms and Conditions of Webb"
```

**Review by dutterbutter — CHANGES_REQUESTED** [review]
A couple changes but lgtm

**Review by dutterbutter — APPROVED** [review]

**Pull request merged**